### PR TITLE
DEVDOCS-4093 Update JWT example.

### PR DESCRIPTION
**Pull request by bc-tgomez**
# [DEVDOCS-4093](https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4093)

## Update JWT example and include response
* Include newest JWT example and added an extra tab including the response

### CODE REVIEW

**Review by slsriehl — CHANGES_REQUESTED** [review]
@bc-tgomez link to the auth doc, don't duplicate. in the paragraph that follows the link, note that the response is plain text and that the JWT code block is decoded.

**Comment by bc-tgomez** [comment]
> @bc-tgomez link to the auth doc, don't duplicate. in the paragraph that follows the link, note that the response is plain text and that the JWT code block is decoded.

Converting to draft to work on those changes.

**Comment by slsriehl** [comment]
@bc-tgomez how do you feel about it now?

**Comment by bc-tgomez** [comment]
@slsriehl Thanks so much for the edits! Looks great to me.

**Review by traci707 — APPROVED** [review]
LGTM!

File: docs/api-docs/customers/current-customer-api.md [hunk 1]
```diff
@@ -1,60 +1,30 @@
...
 
-## Identifying logged-in customers securely
+To address this need, BigCommerce provides a Current Customer endpoint that your app can access on the storefront using JavaScript. This endpoint allows a remote application, such as a third-party subscription billing app, to return a JSON web token, or _JWT_, with identifying customer details. The information is signed with your [app API account's client secret](/api-docs/getting-started/authentication/rest-api-authentication#api-accounts).
 
-Suppose your application interacts dynamically with the BigCommerce storefront and conveys specific information to a particular logged-in customer. You must confirm that customer's identity within the insecure environment of the user's browser before revealing any sensitive information.
+<!-- theme: info -->
+> #### API account notes
+> - This endpoint requires **app API account** credentials. For more information about generating accounts, consult our [Guide to API Accounts](/api-docs/getting-started/authentication/rest-api-authentication#app-api-accounts).
```
Comment (stanzikratelbc) [inline docs/api-docs/customers/current-customer-api.md:11]: replace "our" with "the"

File: docs/api-docs/customers/current-customer-api.md [hunk 2]
```diff
@@ -1,60 +1,30 @@
...
-      true
-    );
-    xmlhttp.send();
-  }
-  customerJWT();
-</script>
-```
+## Request
+
+To test this endpoint, save this [Current Customer API example request](/api-docs/getting-started/authentication#current-customer-api-example-request) JavaScript snippet in the **Script Manager** of one of your sandbox store's Stencil storefronts. Include your app API account's client ID as the value of the `app_client_id` query parameter.
```
Comment (stanzikratelbc) [inline docs/api-docs/customers/current-customer-api.md:17]: I would change to: "To test the Current Customer endpoint..." Consider adding either a link to Script Manager docs or a screenshot

File: docs/api-docs/customers/current-customer-api.md [hunk 3]
```diff
@@ -1,60 +1,30 @@
...
 
-The above JavaScript should alert the browser with a JWT token after logging into the storefront with a customer account. The JWT returned from this endpoint (example below) can be decoded on [JWT.IO](https://jwt.io/).
+## Response
 
-If a shopper is browsing as a guest, BigCommerce will return a `404` response, and you will see an error message. Wrapping your script in a `{{#if customer}}` Handlebars helper will check for a logged-in customer before running the request. For more information, visit Handlebars Helpers. [Handlebars Helpers](/stencil-docs/reference-docs/handlebars-helpers-reference#if).
```
Comment (stanzikratelbc) [inline docs/api-docs/customers/current-customer-api.md:53]: replace "logged-in" with "signed-in"

**Review by stanzikratelbc — COMMENTED** [review]
Added some review comments, main recommendation is change "log in" to "sign in"

File: docs/api-docs/customers/current-customer-api.md [hunk 4]
```diff
@@ -1,60 +1,30 @@
...
-> - An app client ID is required in requests to `/customer/current.jwt`.
-> - To generate an app client ID, create an app in the [BigCommerce Developer Portal](https://devtools.bigcommerce.com/).
-> - Use the app's secret to validate the signature on the JWT.
-> - The app doesn't need to be installed or published on a store to use the client ID to get the JWT.
-
-
-
-## Example JavaScript
-
-Below is an example JavaScript code snippet that will access this JWT. To test the JWT functionality, you can install this JavaScript on your sandbox BigCommerce store. You must include your application's client ID in the request to identify the requesting application.
```
[the remark this answers is not in the document]
Comment (bc-tgomez) [reply docs/api-docs/customers/current-customer-api.md:22]: Thanks! This was removed from the article but I'll keep this in mind.

File: docs/api-docs/customers/current-customer-api.md [hunk 5]
```diff
@@ -1,60 +1,30 @@
...
-    xmlhttp.send();
-  }
-  customerJWT();
-</script>
-```
+## Request
+
+To test this endpoint, save this [Current Customer API example request](/api-docs/getting-started/authentication#current-customer-api-example-request) JavaScript snippet in the **Script Manager** of one of your sandbox store's Stencil storefronts. Include your app API account's client ID as the value of the `app_client_id` query parameter.
 
-The above JavaScript should alert the browser with a JWT token after logging into the storefront with a customer account. The JWT returned from this endpoint (example below) can be decoded on [JWT.IO](https://jwt.io/).
```
[the remark this answers is not in the document]
Comment (bc-tgomez) [reply docs/api-docs/customers/current-customer-api.md:51]: Thanks! This was removed from the article but I'll keep this in mind.

File: docs/api-docs/customers/current-customer-api.md (hunk 1 above)
Comment (bc-tgomez) [reply docs/api-docs/customers/current-customer-api.md]: Done, thanks!

File: docs/api-docs/customers/current-customer-api.md (hunk 2 above)
Comment (slsriehl) [reply docs/api-docs/customers/current-customer-api.md:17]: the issue with screenshots of script manager is that it's different in MSF vs SSF. the DevDocs we have about scripts are about the API. I'm suggesting a link to KB for the UI docs on script manager, which should be kept up to date better than this.

```suggestion
To test this endpoint, save this [Current Customer API example request](/api-docs/getting-started/authentication#current-customer-api-example-request) JavaScript snippet in the storefront's **Script Manager**, located in the store control panel. Include your app API account's client ID as the value of the `app_client_id` query parameter. 

For more about adding scripts with the UI, see our support article on [Using Script Manager](https://support.bigcommerce.com/s/article/Using-Script-Manager).

```

File: docs/api-docs/customers/current-customer-api.md [hunk 6]
```diff
@@ -1,60 +1,33 @@
 # Current Customer API
 
+## Securely identifying signed-in customers
 
+Suppose your application interacts dynamically with a Stencil storefront and conveys specific information to a particular signed-in customer. You must confirm that customer's identity within the insecure environment of the user's browser before revealing any sensitive information.
```
Comment (slsriehl) [inline docs/api-docs/customers/current-customer-api.md:5]:
```suggestion
Suppose your application interacts dynamically with a storefront and conveys specific information to a particular signed-in customer. You must confirm that customer's identity within the insecure environment of the user's browser before revealing any sensitive information.

```

File: docs/api-docs/customers/current-customer-api.md [hunk 7]
```diff
@@ -1,60 +1,33 @@
 # Current Customer API
 
+## Securely identifying signed-in customers
 
+Suppose your application interacts dynamically with a Stencil storefront and conveys specific information to a particular signed-in customer. You must confirm that customer's identity within the insecure environment of the user's browser before revealing any sensitive information.
 
-## Identifying logged-in customers securely
+To address this need, BigCommerce provides a Current Customer endpoint that your app can access on the storefront using JavaScript. This endpoint allows a remote application, such as a third-party subscription billing app, to return a JSON web token, or _JWT_, with identifying customer details. The information is signed with your [app API account's client secret](/api-docs/getting-started/authentication/rest-api-authentication#api-accounts).
```
Comment (slsriehl) [inline docs/api-docs/customers/current-customer-api.md:7]:
```suggestion
To address this need, BigCommerce provides a Current Customer endpoint that your app can access on the storefront using JavaScript. This endpoint allows a remote application, such as a third-party subscription billing app, to request a JSON web token, or _JWT_, with identifying customer details. The response is encrypted with your [app API account's client secret](/api-docs/getting-started/authentication/rest-api-authentication#api-accounts).

```

**Pull request merged**